_docs/node-operator-requirements.md

```diff
@@ -0,0 +1,153 @@
+---
+title: Node Operator Requirements
+subtitle: Technical infrastructure requirements to run your node
+author: irsal
+tags: [infra]
+---
+
+## Node Infrastructure Requirements
+On our path to decentralization and trustlessness, collators are a very important piece of the puzzle. We must maintain a high level of performance (as close to a 12 second block production time as possible), and a high degree of availability. As such, the Foundation must prescribe certain technical specifications when onboarding node operators. 
+Bare metal over cloud, especially when costs are a consideration to the node operator.
+- High internet connection (5 gigabits per second).
+- At least 8 CPU cores (fastest core speed).
+- At least 16GB of RAM.
+- At least 1TB of storage.
+- Monitoring and alerting in place (Prometheus, Grafana, the like).
+
+Please sync a few days before your intended collation / block production candidacy to sync the nodes. Once your node is synced, you will find that the block numbers are up to date with both the relay chain and the parachain. Check out the [Telemetry](https://telemetry.polkadot.io/#list/0x0f62b701fb12d02237a33b84818c11f621653d2b1614c777973babf4652b535d) site for more information.
+
+## Ports
+@nikhil TODO
+Please ensure the following configuration for your node.
+- P2P port must be open to incoming traffic:
+    - Source: Any
+    - Destination: 30333, 30334 TCP
+- RPC port is closed down. (In other words, only you can do rpc calls.)
```

Comment:
I removed this. By default as long as you're not adding the `--rpc-external` flag the node will not accept non-local rpc requests. One less thing node operators can worry about.